src/server/pfs/server/driver.go

```diff
@@ -2216,6 +2221,18 @@ func (d *driver) globFile(ctx context.Context, commit *pfs.Commit, pattern strin
 		return nil, err
 	}
 
+	if !isGlob(pattern) {
+		node, err := tree.Get(pattern)
+		// Path not found is a no-op
+		if hashtree.Code(err) == hashtree.PathNotFound {
+			return []*pfs.FileInfo{}, nil
```

Comment:
This can just be, return `nil, nil`. I don't think there's a meaningful difference between an empty slice and `nil` slice.